esp/services/ws_store/espstorelib/daliKVStore.cpp

```diff
@@ -35,7 +35,7 @@ bool CDALIKVStore::createStore(const char * apptype, const char * storename, con
     VStringBuffer xpath("Store[%s='%s'][1]", DALI_KVSTORE_NAME_ATT,  storename);
     if (root->hasProp(xpath.str()))
     {
-        WARNLOG("DALI KV Store: Cannot create app '%s' entry, it already exists", storename);
+        IWARNLOG("DALI KV Store: Cannot create app '%s' entry, it already exists", storename);
```

Comment:
This is either an operator and/or user issue. I assume the I in IWARNLOG means internal?
I fear the meaning of the single letter prefix might get glossed over in the future, can we consider a more explicit form of these calls? INTERNAL_WARNLOG, etc?

Reply:
@rpastrana It's a good point. First, I'd like to point out that the function names are described in detail in system/jlib/jlog.hpp: lines 38-111.  

As per your point, over time the purposes for the original functions were forgotten and a few  of the wrapper logging functions (WARNLOG, ERRLOG and DBGLOG) were used for everything regardless of audience.  For example, you'll see WARNLOG and ERRLOG used everywhere although these were for user messages only.

We deliberately added a prefix letter so that developers would be encouraged to read the internal documentation to fully understand which one to use   It's more difficult now to simply "guess" which one to use without reading the documentation.   This is important as there is more to logging that just the few common ones we see a lot. There's a few important logging functions that are rare but should be used where appropriate: for example, DISLOG for disasters and there are plenty of combination of Audience+Class+LogMsgCode for which there are no wrapper classes.

**Furthermore**: eventually, the plan is to remove these wrapper functions and have the underlying call used everywhere. We are using these new wrapper functions temporarilly to quickly update all the messages so the audience is correct.

Reply:
@shamser thanks for the detailed response. I'm very certain the single character prefix isn't going to prompt a typical developer to look up the detailed description, as you pointed out a lot of copy/paste is going to happen. If you expand the single char prefix to the full word it represents, it'll remove the guessing out of it and will cut down on misuse. Is there a technical reason keeping us from expanding the prefix?

Reply:
@shamser if the amount of lines affected by renaming the calls is the only thing holding us back, I recommend we do it, perhaps as part of a separate issue if needed. Btw, eclipse, etc should allow us to rename the functions and all of its references (and in some cases related comments) automatically.

Reply:
@ghalliday Can we have your thoughts on this?  Should the functions be renamed?

Reply:
Not as part of this PR.

We have split this change up into phases - and into components - for a reason. Renaming the functions cannot be done until all the components PRs have been accepted.